rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-import { Route } from 'react-router-dom';
+import { Route, useLocation, Redirect } from 'react-router-dom';
+import validatePath from '../../utils/validatePath';
 
 import routePaths from '../../constants/routePaths';
 import { Rig } from '../index';
 import NavBar from '../../components/navBar/NavBar'
 
 const Landing = () => {
+  const { pathname } = useLocation();
+  const regex = /^[a-z0-9]+$/i;
+  const pathInclusion = routePaths.landing + '/rig';
+  const pathParam = pathname.slice(13);
+
+  const isValidRigPath = validatePath(
+    pathname,
+    pathInclusion,
+    pathParam,
+    regex
+  );
+
   return (
     <div>
       <NavBar />
       <h1>Landing Works!</h1>
+      {!isValidRigPath && <Redirect to={routePaths.landing} />}
```

Comment:
Instead of adding a line to the jsx we can programmatically navigate the user to '/landing'

Reply:
added history push instead